<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChangeDig</class>
 <widget class="QWidget" name="PasswordChangeDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label
{
	background-color : rgb(245,245,245);
	color : black
}</string>
     </property>
     <property name="text">
      <string>USER ACCOUNT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widgetContent" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widgetContent
{
	background-color : white;
	color : black
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="edtConfirmPassword">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit#edtConfirmPassword
{
	background : rgb(249,249,249);
	border: 1px solid gray;
	border-radius : 5px
}</string>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="edtPassword">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit#edtPassword
{
	background : rgb(249,249,249);
	border: 1px solid gray;
	border-radius : 5px
}</string>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnCancel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btnCancel
{
	background : rgb(181,186,194);;
	color :White;
	border-radius : 5px;
}</string>
        </property>
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnOKPassword">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#btnOKPassword
{
	background : rgb(236,61,64);;
	color :White;
	border-radius : 5px;
}</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
